Sources_v3/Controllers/ChannelController/ChannelController_Tests.swift

```diff
@@ -310,6 +310,9 @@ class ChannelController_Tests: StressTestCase {
 
         // Simulate `startUpdating` call
         controller.startUpdating()
+        
+        // Simulate updater's channelCreatedCallback call
+        env.channelUpdater!.update_channelCreatedCallback!(channelId)
```

Comment:
This test was actually wrong, and when I fixed the ChannelUpdater this test failed.
ChannelController shouldn't have the channel object before `channelCreatedCallback` is called from its worker.